InTune/InTune/FirebaseServices/DatabaseService.swift

```diff
@@ -10,36 +10,167 @@ import Foundation
 import Firebase
 import FirebaseFirestore
 import FirebaseAuth
+
+
 class DatabaseService {
+  
   static let artistsCollection = "artists"
-  static let usersCollection = "users"
+    //rename to users
+    static let artistPosts = "artistPost"
+    static let favCollection = "favoriteArtists"
+    static let gigPosts = "gigPosts"
+    
     private let db = Firestore.firestore()
   static let shared = DatabaseService()
     
-  public func createArtist(artist: Artist, authDataResult: AuthDataResult, completion: @escaping (Result<Bool,Error>) -> ()){
+    //rename this to users
+  public func createArtist(authDataResult: AuthDataResult, completion: @escaping (Result<Bool,Error>) -> ()){
     guard let email = authDataResult.user.email else {return}
-    db.collection(DatabaseService.artistsCollection).document(authDataResult.user.uid).setData(["name": artist.name, "email": email, "tags": artist.tags, "instruments": artist.instruments, "city": artist.city]){ (error) in
+    db.collection(DatabaseService.artistsCollection).document(authDataResult.user.uid).setData(["email": email, "artistId": authDataResult.user.uid, "createdDate": Timestamp()]){ (error) in
       if let error = error {
         completion(.failure(error))
       } else {
         completion(.success(true))
       }
     }
   }
-//  public func createMedia(imediaName: String, displayName: String, completion: @escaping (Result<String,Error>) ->()) {
-//    guard let user = Auth.auth().currentUser else { return }
-//    let documentRef = db.collection(DatabaseService.itemsCollection).document()
-//    db.collection(DatabaseService.itemsCollection).document(documentRef.documentID).setData(["itemName" :itemName,
-//                                                 "details":details,
-//                                                 "id":documentRef.documentID,
-//                                                 "listedDate":Timestamp(date: Date()),
-//                                                 "postedBy":displayName,
-//                                                 "postedById":user.uid]) { (error) in
-//                                                  if let error = error {
-//                                                    completion(.failure(error ))
-//                                                  } else {
-//                                                    completion(.success(documentRef.documentID))
-//                                                  }
-//    }
-//  }
+    
+    
+    
+    //update function for user experience ( isAnArtist == true )
```

Comment:
follow this for generating comments for a function

https://stackoverflow.com/questions/8954087/auto-generate-method-comment-in-xcode

<img width="602" alt="Screen Shot 2020-06-13 at 2 56 16 PM" src="https://user-images.githubusercontent.com/5465858/84579827-18b59e00-ad86-11ea-880b-8792771a36a2.png">